django_tables2/columns/base.py

```diff
@@ -239,6 +239,18 @@ def render(self, value):
         """
         return value
 
+    def order(self, queryset, is_descending):
+        """
+        Returns the queryset of the table.
+
+        This method can be overridden by :ref:`table.order_FOO` methods on the
```

Comment:
`table.order_FOO` is a placeholder name, so no, it doesn't actually point to a real function. Would you like me to rephrase it?